server/views.py

```diff
@@ -4,16 +4,20 @@
 from django.shortcuts import (get_object_or_404,
                               redirect,
                               render)
+from social_django.models import UserSocialAuth
 
 from .notebooks.models import Notebook, NotebookRevision
 from .base.models import User
 
 
 def get_user_info_dict(user):
     if user.is_authenticated:
+        user_social_auth = UserSocialAuth.objects.get(user=user)
+        social_auth_extra_data = user_social_auth.extra_data
         return {
-            'name': user.username,
+            'name': social_auth_extra_data['login'],
```

Comment:
Any reason for this change? I believe it will break if/when we ever allow non-social-auth logins.

Reply:
True, not sure what happened here!